sos/report/plugins/navicli.py

```diff
@@ -58,29 +60,10 @@ def get_navicli_SP_info(self, SP_address):
     def setup(self):
         self.get_navicli_config()
         CLARiiON_IP_address_list = []
-        CLARiiON_IP_loop = "stay_in"
-        while CLARiiON_IP_loop == "stay_in":
-            try:
-                ans = input("CLARiiON SP IP Address or [Enter] to exit: ")
-            except Exception:
-                return
-            if self.exec_cmd("navicli -h %s getsptime" % (ans,))['status']:
-                CLARiiON_IP_address_list.append(ans)
-            else:
-                if ans != "":
-                    print("The IP address you entered, %s, is not to an "
-                          "active CLARiiON SP." % ans)
-                if ans == "":
-                    CLARiiON_IP_loop = "get_out"
-        # Sort and dedup the list of CLARiiON IP Addresses
-        CLARiiON_IP_address_list.sort()
+        for addr in set(self.get_option("ipaddrs").split()):
+            if self.exec_cmd("navicli -h %s getsptime" % (addr))['status']:
```

Comment:
Should this be `['status'] == 0`?

Reply:
Ah sure. Blind copying previous test (that didnt work for years..). Thx for spotting it.